scheds/rust/scx_chaos/src/lib.rs

```diff
@@ -192,6 +193,10 @@ impl Builder<'_> {
         // this codepath
         open_skel.maps.rodata_data.select_idle_in_enqueue = false;
 
+        if self.p2dq_opts.queued_wakeup {
+            open_skel.struct_ops.chaos_mut().flags |= *compat::SCX_OPS_ALLOW_QUEUED_WAKEUP;
```

Comment:
Is this intended? In that case maybe mention in the commit message that we're enabling this also in scx_chaos.

Reply:
Definitely intended, chaos maintains all the p2dq flags that it can so it makes sense for this to be included there too. I don't mind if it's documented or not, it will automatically be included in the `--help`.